### PR TITLE
MADE word of the day working using javascript

### PR DESCRIPTION
**Pull request by Ash914027**
### Description of Changes ✨

- **Feature Added**: 
  Implemented a "Word of the Day" feature that displays a daily quote along with an associated image. Users can click a button to cycle through different quotes, enhancing engagement and motivation.

- **Bug Fix**: 
  Resolved an issue where images did not load correctly due to incorrect file paths. Ensured all image paths are valid and accessible.

- **UI Improvement**: 
  Enhanced the user interface by styling the layout for better readability and visual appeal. Added CSS for responsive design, ensuring the feature looks good on various devices. Adjusted font sizes and margins for improved spacing and alignment.

---

download (29)

### CODE REVIEW

**Comment by Ash914027** [comment]
PLEASE MERGE THIS PULL REQUEST

**Comment by ANSHIKA-26** [comment]
issue number ?

File: index.html [hunk 1]
```diff
@@ -1075,12 +1091,16 @@ <h2 class="accordion__question">How can I stay updated with new blog posts?</h2>
...
 </section>
 <!-- Word of the Day Section -->
-<div id="word-of-the-day" style="background-color: rgb(248, 181, 181); padding: 20px; border-radius: 5px; color: black;">
-    <h2>Word of the Day</h2>
-    <p id="word"></p>
-    <p id="definition"></p>
+<div id="wordContainer">
+    <div id="word">Word of the Day</div>
+    <div id="definition">Definition will appear here.</div>
+    <img id="image" src="" alt="Word Image">
```
Comment (ANSHIKA-26) [inline index.html:1097]: what exactly are you trying to do here ? there is no image in the src 
what do you wish to display ?

**Review by ANSHIKA-26 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (ANSHIKA-26) [reply index.html:1097]: image
this is how it looks like , fix this

**Comment by ANSHIKA-26** [comment]
where is the word of the day ? you just gave it the title word of the day without any word or definition ?

**Comment by ANSHIKA-26** [comment]
image
see , here you can see the word abundance with its meaning , its not visible in your pr

File: index.html (hunk 1 above)
Comment (Ash914027) [reply index.html:1097]: Ok I will fix it

**Comment by ANSHIKA-26** [comment]
closing due to no reply

**Pull request closed without merging**